app/pages/learn/01_tutorial/04_mastering-the-api/02_invoke/00_methodhandle.md

```diff
@@ -0,0 +1,531 @@
+---
+id: api.invoke
+title: "Introduction to Method Handles"
+slug: learn/introduction_to_method_handles
+type: tutorial
+category: api
+category_order: 1
+layout: learn/tutorial.html
+subheader_select: tutorials
+main_css_id: learn
+description: "What is Method Handle mechanism, how is it different from Reflection API, and what tooling does it provide."
+author: ["NataliiaDziubenko"]
+toc:
+  - What are Method Handles {intro}
+  - Access checking {access}
+  - Method handle lookup {lookup}
+  - Method type {methodtype}
+  - Method handle invocation {invocation}
+  - Accessing fields {fields}
+  - Working with arrays {arrays}
+  - Method handles vs Reflection {vsreflection}
+  - Conversion between Reflection API and method handles {unreflect}
+  - Exception handling {exceptions}
+  - Method handle transformations {transformations}
+  - Conclusion {conclusion}
+last_update: 2024-04-29
+---
+
+<a id="intro">&nbsp;</a>
+## What are Method Handles
+Method Handles are a low level mechanism used for method lookup and invocation. It is often compared to reflection, but in fact, these two APIs serve different purposes. 
+There are some overlapping capabilities that both Reflection API and Method Handles can provide. 
+For example, method invocation, where Method Handles perform better.
+
+What exactly is a Method Handle? It's a direct reference to a method, constructor, or field, which can be invoked. 
+Method Handle API allows manipulations on top of simple pointer to the method, that allow us to insert or reorder the arguments, or transform the return values, for example.
+
+Let's take a closer look at what Method Handle mechanism can provide and how we can effectively use it.
+
+<a id="access">&nbsp;</a>
+## Access checking
+The access checking for method handle invocations is done differently compared to the Reflection API. 
+With reflection, each call results in access checks for the caller. 
+For method handles, the access is only checked when the method handle is created. 
+The access level that it has can't exceed the access level of the class where it's created; however, it can be further limited.
```

Comment:
While reading _The access level that it has can't exceed_ I do not understant to what _it_ reffers to. Maybe you can rephrase this sentence for clarity?
Thank you